src/js/background/background.js

```diff
@@ -1,8 +1,13 @@
-const RELAY_SITE_ORIGIN = "http://127.0.0.1:8000";
+function startupInit() {
+  const RELAY_SITE_ORIGIN = "http://127.0.0.1:8000";  
+  browser.storage.local.set({ RELAY_SITE_ORIGIN });
+  browser.storage.local.set({ maxNumAliases: 5 });
+  browser.storage.local.set({ relaySiteOrigin: RELAY_SITE_ORIGIN });
+  browser.storage.local.set({ relayApiSource: `${RELAY_SITE_ORIGIN}/api/v1` });
```

Comment:
Ignorant Q: But do these have to be set separately, or can we set one object w/ multiple keys?
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/set implies we can almost condense these four `set()` calls into a single call w/ object w/ 4 keys. 🤷

Reply:
We can definitely consolidate! Filed #538